examples/se2_inverse.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# This example will use torch to solve min_x1 || x1^-1 - x2 ||^2,
+# where x1 and x2 are SE2 objects
+
+import torch
+import theseus as th
+from theseus import LieGroupTensor
+from theseus.geometry.lie_group import LieGroup
+
+from theseus.geometry.tests.test_se2 import create_random_se2
+
+rng = torch.Generator()
+rng.manual_seed(0)
+x1 = create_random_se2(1, rng)
+x2 = create_random_se2(1, rng)
+
+
+def run(x1: LieGroup, x2: LieGroup, num_iters=10, use_lie_tangent=True):
+    x1.data = LieGroupTensor(x1)
```

Comment:
We don't need this line if `theseus.LieGroup.data` is changed from `torch.Tensor` to `theseus.LieGroupTensor`.

Reply:
I think this should be fine, but let's wait until we finalize the implementation of `LieGroupTensor`.